lldb/include/lldb/API/SBTarget.h

```diff
@@ -184,6 +184,8 @@ class LLDB_API SBTarget {
 
   SBProcess LoadCore(const char *core_file);
   SBProcess LoadCore(const char *core_file, lldb::SBError &error);
+  SBProcess LoadCore(SBFile &file);
```

Comment:
Did you add this for consistency? I don't think we should be adding an overload without error handling.

Reply:
+1, please don't add a new API that gives no feedback about what may have gone wrong.

Reply:
Just add the error to your function call:
```
SBProcess LoadCore(SBFile &file, lldb::SBError &error);
```
The first `SBProcess LoadCore(const char *core_file);` should be marked as deprecated since we want people to use `SBProcess LoadCore(const char *core_file, lldb::SBError &error);` so they get error information back. In our public API we never remove an older API, and that is why `SBProcess LoadCore(const char *core_file);` is still there.